src/cmd/ksh93/tests/modifiers.sh

```diff
@@ -94,3 +94,22 @@ do
         fi
     done
 done
+
+# ==========
+# https://github.com/att/ast/issues/70
+cat > $TEST_DIR/modifier-in-loop.sh <<EOF
```

Comment:
FWIW, This scenario is why I introduced the concept of unit test sub directories. For example, *src/cmd/ksh93/tests/signal/*. Whether or not we need a different mechanism for organizing tests within a unit test module is TBD. Don't refactor this as it is okay and consistent with 99.9% of the test code. I'm just commenting to get people to think about what we should do to improve the ksh unit test framework.